ballerina-test/src/test/resources/kubernetes/hello_world_k8s.bal

```diff
@@ -0,0 +1,23 @@
+import ballerina/http;
```

Comment:
Some of bal files has LICENSE, shall we add here as well?

Reply:
https://github.com/ballerina-platform/ballerina-distribution/pull/444/commits/96b01c79136f64c90671e0b7151493c75c047704